### PR TITLE
Dump drogon to v1.8.2 and trantor to v1.5.8 and c-ares to 1.18.1

### PR DESCRIPTION
**Pull request by TES286**
Dump drogon to v1.8.2 and trantor to v1.5.8

### CODE REVIEW

**Comment by TES286** [comment]
I've looked at the checks that fail, and some of the checks that fail on Windows are interesting: file 'C:/Users/runneradmin/AppData/Local' not found
Windows uses '\' to separate paths, but here it becomes '/', which is the usage of Linux and Mac. But in fact, this path will almost always exist, and many programming languages or tools will automatically convert '/' to '\' , but there doesn't seem to be one here, the OS doesn't recognize it.
So I don't think it's a problem with this PR
But why is it okay on Windows 2019? Metaphysics?

**Comment by waruqi** [comment]
> CUSTOMBUILD : error : file 'C:/Users/runneradmin/AppData/Local' not found 

It should be trantor/CMakeLists.txt issue, maybe it has a custom build command, and it does not support path with space better.

you can check it's CMakeLists.txt and add patch to fix it.

**Comment by waruqi** [comment]
or you can try use cmake to build trantor on vs2022 directly and debug it.

**Comment by TES286** [comment]
Trantor can be successfully installed using vcpkg, see https://github.com/TES286-org/test-build/actions/runs/3871779193/jobs/6599887573

But the project itself does not have any patch in vcpkg, only its dependencies have newer versions.

I'll test that updating its dependencies succeeds.

File: packages/c/c-ares/xmake.lua [hunk 1]
```diff
@@ -26,24 +36,5 @@ package("c-ares")
...
-            table.insert(configs, "--enable-debug")
-        else
-            table.insert(configs, "--disable-debug")
-        end
-        import("package.tools.autoconf").install(package, configs)
-    end)
-
-    on_test(function (package)
-        assert(package:has_cfuncs("ares_library_init", {includes = "ares.h"}))
-    end)
```
Comment (waruqi) [inline packages/c/c-ares/xmake.lua:49]: why?

Comment (TES286) [reply packages/c/c-ares/xmake.lua:49]: It's my wrong... sorry....

**Pull request merged**